server/src/server.ts

```diff
@@ -522,11 +522,13 @@ connection.onHover((textDocumentPosition: TextDocumentPositionParams): Hover =>
     let path = pathData.path;
 
     if (path.startsWith('/images/')) {
+        let splitPath = path.split('/');
+        if (splitPath.length > 3) {
+            path = splitPath.slice(0, 3).join('/');
+        }
         let imageData = getFromPath(pathData.jsonMap.data, path);
         if (imageData) {
-            if (!path.endsWith('/uri')) {
-                imageData = imageData.uri;
-            }
+            imageData = imageData.uri;
```

Comment:
This can crash when `uri` is left unspecified, right?  Granted the image is not much use without a uri.

Reply:
It looks like the validator has an error if there isn't a uri too.

Reply:
I see that too.  @lexaknyazev Do you need a new issue for that one?

Reply:
Validator has been fixed in `2.0.0-dev.2.0`.

Reply:
Aha, I had just un-done the validator upgrade to test this PR.  I should have tested the two together!